bakery_lint/fonttests/test_ttf.py

```diff
@@ -664,9 +667,10 @@ def test_metrics_linegaps_are_zero(self):
             _ = '[%s] have not zero linegaps'
             self.fail(_ % ', '.join(fonts_gaps_are_not_zero))
 
+    @tags('required')
     @autofix('bakery_cli.fixers.Vmet')
     def test_metrics_ascents_equal_bbox(self):
-        """ Check that ascents values are same as max glyph point """
+        """ Ascent values are same as max glyph point? """
```

Comment:
Should be

`Vertical Metrics ascent values are the same as the y-max of all glyphs in the family?`

and thus part of the **family** checks.